File: app/src/main/java/cc/smartcash/wallet/Models/FullTransaction.java
package cc.smartcash.wallet.Models;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;

import java.util.ArrayList;

@JsonIgnoreProperties(ignoreUnknown = true)
public class FullTransaction {
    ArrayList<Object> vin = new ArrayList<>();
    ArrayList<Object> vout = new ArrayList<>();
    private String txid;
    private float version;
    private float locktime;
    private String blockhash;
    private float blockheight;
    private float confirmations;
    private long time;
    private long blocktime;
    private float valueOut;
    private float size;
    private float valueIn;
    private float fees;


    // Getter Methods

    public String getTxid() {
        return txid;
    }

    public void setTxid(String txid) {
        this.txid = txid;
    }

    public float getVersion() {
        return version;
    }

    public void setVersion(float version) {
        this.version = version;
    }

    public float getLocktime() {
        return locktime;
    }

    public void setLocktime(float locktime) {
        this.locktime = locktime;
    }

    public String getBlockhash() {
        return blockhash;
    }

    public void setBlockhash(String blockhash) {
        this.blockhash = blockhash;
    }

    public float getBlockheight() {
        return blockheight;
    }

    public void setBlockheight(float blockheight) {
        this.blockheight = blockheight;
    }

    public float getConfirmations() {
        return confirmations;
    }

    public void setConfirmations(float confirmations) {
        this.confirmations = confirmations;
    }

    // Setter Methods

    public long getTime() {
        return time;
    }

    public void setTime(long time) {
        this.time = time;
    }

    public long getBlocktime() {
        return blocktime;
    }

    public void setBlocktime(long blocktime) {
        this.blocktime = blocktime;
    }

    public float getValueOut() {
        return valueOut;
    }

    public void setValueOut(float valueOut) {
        this.valueOut = valueOut;
    }

    public float getSize() {
        return size;
    }

    public void setSize(float size) {
        this.size = size;
    }

    public float getValueIn() {
        return valueIn;
    }

    public void setValueIn(float valueIn) {
        this.valueIn = valueIn;
    }

    public float getFees() {
        return fees;
    }

    public void setFees(float fees) {
        this.fees = fees;
    }
}

File: app/src/main/java/cc/smartcash/wallet/Models/FullTransaction.kt
package cc.smartcash.wallet.Models

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import java.io.Serializable
import java.util.*

@JsonIgnoreProperties(ignoreUnknown = true)
class FullTransaction : Serializable {

    internal var vin = ArrayList<Any>()
    internal var vout = ArrayList<Any>()
    // Getter Methods

    var txid: String? = null
    var version: Float = 0.toFloat()
    var locktime: Float = 0.toFloat()
    var blockhash: String? = null
    var blockheight: Float = 0.toFloat()
    var confirmations: Float = 0.toFloat()
    // Setter Methods

    var time: Long = 0
    var blocktime: Long = 0
    var valueOut: Float = 0.toFloat()
    var size: Float = 0.toFloat()
    var valueIn: Float = 0.toFloat()
    var fees: Float = 0.toFloat()
}